ecommerce/src/main/kotlin/io/hhplus/ecommerce/common/client/MockDataPlatformClient.kt

```
@@ -0,0 +1,102 @@
+                logger.info {
+                    "[DataPlatform] 전송 성공: orderId=${orderInfo.orderId}, orderNumber=${orderInfo.orderNumber}"
+                }
+                DataPlatformResponse(success = true, message = "전송 성공")
+            } else {
+                failureCount.incrementAndGet()
+                logger.warn {
+                    "[DataPlatform] 전송 실패: orderId=${orderInfo.orderId}, status=${response.statusCode}"
+                }
+                DataPlatformResponse(success = false, message = "HTTP 오류: ${response.statusCode}")
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)